lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -136,39 +136,48 @@ class window.Turbolinks
 
   @loadPage: (url, xhr, options = {}) ->
     triggerEvent 'page:receive'
+    response = new TurboGraft.Response(xhr)
+    options.onlyKeys ?= []
+    options.exceptKeys ?= []
```

Comment:
imo this feels a bit weird to set here

Shouldn't it be pretty unlikely now that this is not defined?  What if `isPartialReplace` did `options.exceptKeys?.length`?